src/components/Button/IconButton/IconButton.tsx

```diff
@@ -39,11 +40,19 @@ type IconButtonProps = UnstyledButtonPropsFor<"button"> &
      * As in IndicatorIcon
      */
     indicator?: "default" | "success" | "critical";
-
     /**
      * Whether the button is interactable
      */
     disabled?: boolean;
+    /**
+     * Whether this button triggers a destructive action.
+     * @default false
+     */
+    destructive?: boolean;
+    /**
+     * The label for the button
+     */
+    label?: string;
```

Comment:
the prop says its a label, in reality its a tooltip? Why can't the caller just wrap it in the tooltip like most other components do

Reply:
Because I think that the pair tooltip and `IconButton` is very common. I add it to avoid to add it manually every time. I want to make it easier/less painful to use.

Reply:
@t3chguy You think that this tooltip is too much?

Reply:
No I just think the docs should say its a tooltip, otherwise the caller has no way to know their label won't just be a label, but be a tooltip

Reply:
Yep make sense, I updated it